lib/tulul_stats_bot/tulul_stats_bot.rb

```diff
@@ -11,7 +11,11 @@ def self.start
 
           query = /\/top_(.+)/.match(message.text).captures[0] rescue nil
           query = query.split('@')[0] rescue nil
-          if query && (TululStats::User.fields.keys.reject{ |field| TululStats::User::EXCEPTION.include?(field) } + TululStats::Entity::ENTITY_QUERY + TululStats::IsTime::TIME_QUERY).include?(query)
+          if /^\/last_tulul([@].+)?/.match(message.text.strip)
```

Comment:
I meant, this `(TululStats::User.fields.keys.reject{ |field| TululStats::User::EXCEPTION.include?(field) } + TululStats::Entity::ENTITY_QUERY + TululStats::IsTime::TIME_QUERY).include?(query)` would be longer.

If you okay with current approach, it is fine to merge this.